--- MPI/node_get_nbc.f
+++ MPI/node_get_nbc.f
@@ -14 +14 @@
-c----CAMx v5.30 101223
+c----CAMx v5.30_patch 110303
@@ -40,0 +41,2 @@
+c    --gwilson-- 03/03/11 changed the allocation of the temp buffer
+c
@@ -124,2 +126,2 @@
-         allocate( buffnest( 2*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*nspec ) )
-         nbuff_save = 2*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*nspec
+         allocate( buffnest( 10*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*nspec ) )
+         nbuff_save = 10*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*nspec
@@ -230 +232 @@
-c----CAMx v5.30 101223
+c----CAMx v5.30 101201
@@ -509 +511 @@
-c----CAMx v5.30 101223
+c----CAMx v5.30 101201
@@ -620,2 +622,2 @@
-         allocate( buffnest( 2*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*ntotsp ) )
-         nbuff_save = 2*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*ntotsp
+         allocate( buffnest( 10*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*ntotsp ) )
+         nbuff_save = 10*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*ntotsp
--- MPI/node_send_nbc.f
+++ MPI/node_send_nbc.f
@@ -14 +14 @@
-c----CAMx v5.30 101223
+c----CAMx v5.30_patch 110303
@@ -41,0 +42,2 @@
+c    --gwilson-- 03/03/11 changed the allocation of the temp buffer
+c
@@ -110,2 +112,2 @@
-         allocate( buffnest( 2*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*nspec ) )
-         nbuff_save = 2*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*nspec
+         allocate( buffnest( 10*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*nspec ) )
+         nbuff_save = 10*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*nspec
@@ -199 +201 @@
-c----CAMx v5.30 101223
+c----CAMx v5.30 101201
@@ -381 +383 @@
-c----CAMx v5.30 101223
+c----CAMx v5.30 101201
@@ -479,2 +481,2 @@
-         allocate( buffnest( 2*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*ntotsp ) )
-         nbuff_save = 2*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*ntotsp
+         allocate( buffnest( 10*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*ntotsp ) )
+         nbuff_save = 10*(mmxp(ifm)+mmyp(ifm))*mmzp(ifm)*ntotsp
