--- SOAP/soapdat.f	Sat Dec 10 21:49:30 2011
+++ SOAP/soapdat.f	Wed Feb 29 16:18:33 2012
@@ -14,0 +15,3 @@
+c
+c     In this test, we produce as much SOA as possible
+c
@@ -21 +24 @@
-      data csat     /7.82, 227., 0.726, 136, 3.92, 55.8, 0.0/
+      data csat     /7*0.0/
@@ -23,2 +26,2 @@
-      data deltah   /66800.,66800.,42000.,42000.,75500.,75500.,0.0/
-      data fpoly0   /0.025/
+      data deltah   /7*0.0/
+      data fpoly0   /1.000/
