--- IO_bin/readinp.f	Sat Dec 10 21:49:34 2011
+++ IO_bin/readinp.f	Sat Mar 03 20:58:16 2012
@@ -126,0 +127,6 @@
+      
+c     This is just needed to adjust Temperature (for sensitivity)
+      real :: deltaTemp
+      
+      deltaTemp = -3.0
+      
@@ -287,0 +294,4 @@
+     
+c adjust temp if needed
+      if (deltaTemp .ne. 0) tsnext(i,j) = tsnext(i,j) + deltaTemp
+      
@@ -290,0 +301,4 @@
+     
+c adjust temp if needed
+      if (deltaTemp .ne. 0) tnext(i,j,k) = tnext(i,j,k) + deltaTemp
+      
