--- CAMx/CAMx.f	Sat Dec 10 21:49:36 2011
+++ CAMx/CAMx.f	Fri Mar 09 23:32:05 2012
@@ -82,0 +83,6 @@
+      
+c     Index into Ozone_Offset_PPB and PPM conc
+      integer ozone_offset_index, icol, irow, ilay
+      integer ozone_offset_index_last
+      real ozone_offset_ppm
+      
@@ -90,0 +97,6 @@
+      
+      real ozone_offset_ppb(24)
+            
+c     our static 24 hour array to modify O3
+      data ozone_offset_ppb /-1.0  -5.0 -10.0 -15.0 -20.0 -20.0 -15.0 -10.0 -5.0 0.0 5.0 10.0
+      &                       20.0 20.0  20.0  20.0  20.0  15.0  10.0   0.0  0.0 0.0 0.0 0.0/
@@ -291,0 +304,33 @@
+
+           write(*,'(a20,$)')    'Adj. O3 .....'
+           write(iout,'(a20,$)') 'Adj. O3 .....'
+           
+c-----We add O3 offset but make sure that we remove the last value (otherwise we have build-up!)
+c     Note that we assume O3 to be at position 3 in the internal fields as well as in the average fields
+
+           ozone_offset_index = time/100
+           ozone_offset_index_last = ozone_offset_index - 1
+           
+c     For all hours expect the first, we need to remove the last addition
+           if (ozone_offset_index .gt. 1) then
+           	ozone_offset_ppm = (ozone_offset_ppb(ozone_offset_index) * 1000.) - (ozone_offset_ppb(ozone_offset_index_last) * 1000.)
+           else
+           	ozone_offset_ppm = (ozone_offset_ppb(ozone_offset_index) * 1000.)
+           endif
+           
+c     We loop thorugh conc(1) this holds the concentrations of the coarse domain
+
+           do ilay = 1,nlay(1)
+             do icol = 1,ncol(1)
+               do irow = 1, nrow(1)
+c                 This indexing code is from readcnc.f 
+c                 O3 is in position 3
+                  n3d = icol + ncol(1)*(irow - 1) + ncol(1)*nrow(1)*(ilay - 1)
+                  n4d = n3d + ncol(1)*nrow(1)*nlay(1)*2
+c                we adjust both the instantaneous and the average field
+                 conc(n4d) = conc(n4d) + ozone_offset_ppm
+                 avcnc(n4d) = avcnc(n4d) + ozone_offset_ppm
+               enddo
+             enddo
+           enddo
+
