--- CAMx/CAMx.f	Sat Dec 10 21:49:36 2011
+++ CAMx/CAMx.f	Fri Mar 09 16:29:17 2012
@@ -69,0 +70 @@
+      include 'custom.inc'
@@ -82,0 +84,6 @@
+      
+c     Index into Ozone_Offset_PPB and PPM conc
+      integer ozone_offset_index, icol, irow, ilay
+      integer ozone_offset_index_last
+      real ozone_offset_ppm
+      
@@ -291,0 +299,33 @@
+
+           write(*,'(a20,$)')    'Adj. O3 .....'
+           write(iout,'(a20,$)') 'Adj. O3 .....'
+           
+c-----We add O3 offset but make sure that we remove the last value (otherwise we have build-up!)
+c     Note that we assume O3 to be at position 3 in the internal fields as well as in the average fields
+
+           ozone_offset_index = time/100
+           ozone_offset_index_last = ozone_offset_index - 1
+           
+c     For all hours expect the first, we need to remove the last addition
+           if (ozone_offset_index .gt. 1) then
+           	ozone_offset_ppm = (ozone_offset_ppb(ozone_offset_index) * 1000.) - (ozone_offset_ppb(ozone_offset_index_last) * 1000.)
+           else
+           	ozone_offset_ppm = (ozone_offset_ppb(ozone_offset_index) * 1000.)
+           endif
+           
+c     We loop thorugh conc(1) this holds the concentrations of the coarse domain
+
+           do ilay = 1,nlay(1)
+             do icol = 1,ncol(1)
+               do irow = 1, nrow(1)
+c                 This indexing code is from readcnc.f 
+c                 O3 is in position 3
+                  n3d = icol + ncol(1)*(irow - 1) + ncol(1)*nrow(1)*(ilay - 1)
+                  n4d = n3d + ncol(1)*nrow(1)*nlay(1)*2
+c                we adjust both the instantaneous and the average field
+                 conc(n4d) = conc(n4d) + ozone_offset_ppm
+                 avcnc(n4d) = avcnc(n4d) + ozone_offset_ppm
+               enddo
+             enddo
+           enddo
+
--- Includes/custom.inc	Fri Mar 09 15:05:24 2012
+++ Includes/custom.inc	Fri Mar 09 16:34:48 2012
@@ -0,0 +1,19 @@
+c----CAMx v5.30 101223
+c 
+c     Here, custom flags are included
+c
+c-----------------------------------------------------------------------
+
+
+c
+c-----------------------------------------------------------------------
+c     Variables for customisations:
+c
+c     Ozone_Offset_PPB -- Offset of Ozone to add for each model hour
+c-----------------------------------------------------------------------
+c
+c
+      real ozone_offset_ppb(24)
+      
+      common /custom/ ozone_offset_ppb
+c
\ No newline at end of file
--- SOAP/soapdat.f	Sat Dec 10 21:49:30 2011
+++ SOAP/soapdat.f	Fri Mar 09 16:28:08 2012
@@ -17,0 +18 @@
+      include 'custom.inc'
@@ -26 +27,5 @@
-c
+      
+c     we abuse this for out little ozone experiment
+      data ozone_offset_ppb /0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 
+      &                      0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0 0.0/
+      
