--- CAMx/CAMx.f	Sat Dec 10 21:49:36 2011
+++ CAMx/CAMx.f	Fri Mar 09 13:31:16 2012
@@ -70,0 +71,2 @@
+c     Needed to modify ozone
+      include 'namelist.inc'
@@ -82,0 +85,6 @@
+      
+c     Index into Ozone_Offset_PPB and PPM conc
+      integer ozone_offset_index, icol, irow, ilay
+      integer ozone_offset_index_last
+      real ozone_offset_ppm
+      
@@ -291,0 +300,28 @@
+
+           write(*,'(a20,$)')    'Adj. O3 .....'
+           write(iout,'(a20,$)') 'Adj. O3 .....'
+           
+c-----We add O3 offset but make sure that we remove the last value (otherwise we have build-up!)
+c     Note that we assume O3 to be at position 3!
+
+           ozone_offset_index = time/100
+           ozone_offset_index_last = ozone_offset_index - 1
+           
+c     For all hours expect the first, we need to remove the last addition
+           if (ozone_offset_index .gt. 1) then
+           	ozone_offset_ppm = (Ozone_Offset_PPB(ozone_offset_index) * 1000.) - (Ozone_Offset_PPB(ozone_offset_index_last) * 1000.)
+           else
+           	ozone_offset_ppm = (Ozone_Offset_PPB(ozone_offset_index) * 1000.)
+           endif
+           
+c     We loop thorugh conc(1) this holds the concentrations of the coarse domain
+
+           do ilay = 1,nlay(1)
+             do icol = 1,ncol(1)
+               do irow = 1, nrow(1)
+                 conc(icol,irow,ilay,3) = conc(icol,irow,ilay,3) + ozone_offset_ppm
+               enddo
+             enddo
+           enddo
+
+
--- CAMx/readnml.f	Sat Dec 10 21:49:37 2011
+++ CAMx/readnml.f	Fri Mar 09 11:48:03 2012
@@ -84 +84,2 @@
-     & Dry_Deposition
+     & Dry_Deposition,
+     & Ozone_Offset_PPB
@@ -192,0 +194 @@
+
--- Includes/camx.inc	Sat Dec 10 21:49:32 2011
+++ Includes/camx.inc	Fri Mar 09 11:40:09 2012
@@ -85,0 +86,11 @@
+c
+c-----------------------------------------------------------------------
+c     Variables for customisations:
+c
+c     Ozone_Offset_PPB -- Offset of Ozone to add for each model hour
+c-----------------------------------------------------------------------
+c
+      real   Ozone_Offset_PPB(24)
+c
+      common /custom/ Ozone_Offset_PPB
+c
\ No newline at end of file
--- Includes/namelist.inc	Sat Dec 10 21:49:32 2011
+++ Includes/namelist.inc	Fri Mar 09 11:47:46 2012
@@ -109,0 +110,2 @@
+c     Added by DCO to allow increase in O3
+      real          Ozone_Offset_PPB(24)
