--- Makefile	Wed Sep 30 14:41:41 2009
+++ Makefile	Wed Sep 30 14:41:41 2009
@@ -268,18 +268,18 @@
 	make modules \
 	   FC=pgf90 \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="$(INCLUDES) -O2 -pc 64 -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic"
+	   FLGS="$(INCLUDES) -O2 -pc 64 -Mdalign -mcmodel=medium -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
 	make hdflib \
 	   HDF=$(HDF) \
 	   FC=pgf90 \
 	   CC=gcc \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -pc 64 -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic"
+	   FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -pc 64 -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
 	make model \
 	   FC=pgf90 \
 	   TARGT=CAMx.$(DOMAIN).$(TARGT_STRING).pg_linux \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="$(INCLUDES) -O2 -pc 64 -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic"
+	   FLGS="$(INCLUDES) -O2 -pc 64 -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
 
 pg_linuxomp:
 	@rm -f $(CAMx_INC)/camx.prm
@@ -287,18 +287,18 @@
 	make modules \
 	   FC=pgf90 \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -mp -O2 -pc 64 -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic"
+	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -mp -O2 -pc 64 -Mconcur=nonuma -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
 	make hdflib \
 	   HDF=$(HDF) \
 	   FC=pgf90 \
 	   CC=gcc \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -pc 64 -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic"
+	   FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -pc 64 -Mconcur=nonuma -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
 	make model \
 	   FC=pgf90 \
 	   TARGT=CAMx.$(DOMAIN).$(TARGT_STRING).pg_linuxomp \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -mp -O2 -pc 64 -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic"
+	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -mp -O2 -pc 64 -Mconcur=nonuma -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
 
 i_linux:
 	@rm -f $(CAMx_INC)/camx.prm
