Index: Makefile.pg_64bit
===================================================================
--- Makefile	(revision 1847)
+++ Makefile	(working copy)
@@ -38,8 +38,6 @@
 #-------------------------------------------------------------------------------
 #
 
-MPI_INST = /usr/local/mpich
-
 .SUFFIXES: .c .f .o .f90 .F90
 
 #ifndef DOMAIN
@@ -74,8 +72,8 @@
 
 ifeq ($(MPI),true)
     MPI_INC   = /usr/local/mpich/include
-    MPI_LIBS = -L$(CAMX_MPI)/util -lparlib \
-        -L$(MPI_INST)/lib -lmpich
+    MPI_LIBS =  -lparlib \
+         -lmpich
     TARGT_STRING = MPI
 else
     MPI_INC   = ./Dummy
@@ -85,7 +83,7 @@
 endif
 
 ifeq ($(HDF),true)
-   HDF_LIB  = -L/usr/lib -L/usr/local/lib -lhdf5 -lz
+   HDF_LIB  = -lhdf5 -lz
    HDF_INC  = /usr/local/include/hdf5
    DEFHDF   = -DUSEHDF
    INCLUDES = -I$(CAMx_INC) -I$(MPI_INC) -I$(IOHDF) -I$(HDF_INC)
@@ -249,18 +247,18 @@
 	make modules \
 	   FC=xlf \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -O2 -static -brename:.flush,.flush_ -brename:.etime,.etime_ -brename:.dtime,.dtime_ -brename:.fdate,.fdate_"
+	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -O2 -brename:.flush,.flush_ -brename:.etime,.etime_ -brename:.dtime,.dtime_ -brename:.fdate,.fdate_"
 	make hdflib \
 	   HDF=$(HDF) \
 	   FC=xlf \
 	   CC=cc \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="$(INCLUDES) -DIBM $(DEFHDF) -O2 -static -brename:.flush,.flush_ -brename:.etime,.etime_ -brename:.dtime,.dtime_ -brename:.fdate,.fdate_"
+	   FLGS="$(INCLUDES) -DIBM $(DEFHDF) -O2 -brename:.flush,.flush_ -brename:.etime,.etime_ -brename:.dtime,.dtime_ -brename:.fdate,.fdate_"
 	make model \
 	   FC=xlf \
 	   TARGT=CAMx.$(DOMAIN).$(TARGT_STRING).ibm \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -O2 -static -brename:.flush,.flush_ -brename:.etime,.etime_ -brename:.dtime,.dtime_ -brename:.fdate,.fdate_"
+	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -O2 -brename:.flush,.flush_ -brename:.etime,.etime_ -brename:.dtime,.dtime_ -brename:.fdate,.fdate_"
 
 pg_linux:
 	@rm -f $(CAMx_INC)/camx.prm
@@ -268,18 +266,18 @@
 	make modules \
 	   FC=pgf90 \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="$(INCLUDES) -O2 -pc 64 -Mdalign -mcmodel=medium -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
+	   FLGS="$(INCLUDES) -O2 -pc 64 -Mdalign -mcmodel=medium -Mextend -Mnoframe -byteswapio"
 	make hdflib \
 	   HDF=$(HDF) \
 	   FC=pgf90 \
 	   CC=gcc \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -pc 64 -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
+	   FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -pc 64 -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio"
 	make model \
 	   FC=pgf90 \
 	   TARGT=CAMx.$(DOMAIN).$(TARGT_STRING).pg_linux \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="$(INCLUDES) -O2 -pc 64 -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
+	   FLGS="$(INCLUDES) -O2 -pc 64 -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio"
 
 pg_linuxomp:
 	@rm -f $(CAMx_INC)/camx.prm
@@ -287,18 +285,18 @@
 	make modules \
 	   FC=pgf90 \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -mp -O2 -pc 64 -Mconcur=nonuma -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
+	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -mp -O2 -pc 64 -Mconcur=nonuma -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio "
 	make hdflib \
 	   HDF=$(HDF) \
 	   FC=pgf90 \
 	   CC=gcc \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -pc 64 -Mconcur=nonuma -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
+	   FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -pc 64 -Mconcur=nonuma -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio"
 	make model \
 	   FC=pgf90 \
 	   TARGT=CAMx.$(DOMAIN).$(TARGT_STRING).pg_linuxomp \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -mp -O2 -pc 64 -Mconcur=nonuma -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic_pgi"
+	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -mp -O2 -pc 64 -Mconcur=nonuma -mcmodel=medium -Mdalign -Mextend -Mnoframe -byteswapio"
 
 i_linux:
 	@rm -f $(CAMx_INC)/camx.prm
@@ -325,13 +323,13 @@
 	make modules \
 	   FC=ifort \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -openmp -O2 -align dcommons -extend_source -convert big_endian -ipo -static"
+	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) -openmp -O2 -align dcommons -extend_source -convert big_endian -ipo"
 	make hdflib \
 	   HDF=$(HDF) \
 	   FC=ifort \
 	   CC=gcc \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -align dcommons -extend_source -convert big_endian -ipo -static"
+	   FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -align dcommons -extend_source -convert big_endian -ipo"
 	make model \
 	   FC=ifort \
 	   TARGT=CAMx.$(DOMAIN).$(TARGT_STRING).i_linuxomp \
@@ -344,18 +342,18 @@
 	make modules \
 	   FC=ifort \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="-I$(CAMx_INC) -V -O3 -m64 -N11 -YEXT_NAMES=LCS -YEXT_SFX=_ -s -YCFRL=1 -W132 -lU77 -L/Applications/Absoft/lib64"
+	   FLGS="-I$(CAMx_INC) -V -O3 -m64 -N11 -YEXT_NAMES=LCS -YEXT_SFX=_ -s -YCFRL=1 -W132 -lU77 "
 	make hdflib \
 	   HDF=$(HDF) \
 	   FC=f90 \
 	   CC=cc \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) $(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O3 -V -m64 -N11 -YEXT_NAMES=LCS -YEXT_SFX=_ -s -YCFRL=1 -W132 -lU77 -L/Applications/Absoft/lib64"
+	   FLGS="-I$(CAMx_INC) -I$(MPI_INC) $(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O3 -V -m64 -N11 -YEXT_NAMES=LCS -YEXT_SFX=_ -s -YCFRL=1 -W132 -lU77 "
 	make model \
 	   FC=ifort \
 	   TARGT=CAMx.$(DOMAIN).$(TARGT_STRING).ab_linux \
            MODULES="-I${MOD_DIR} -module ${MOD_DIR}" \
-	   FLGS="-I$(CAMx_INC) -V -O3 -m64 -N11 -YEXT_NAMES=LCS -YEXT_SFX=_ -s -YCFRL=1 -W132 -lU77 -L/Applications/Absoft/lib64"
+	   FLGS="-I$(CAMx_INC) -V -O3 -m64 -N11 -YEXT_NAMES=LCS -YEXT_SFX=_ -s -YCFRL=1 -W132 -lU77 "
 
 clean:	
 	rm -f $(OBJCTS) $(MOD_OBJCTS) $(IOHDF)/*.o Dummy/*.o *.mod $(MOD_DIR)/*.mod $(CAMX_MPI)/*.o *.a
