--- CAMx/camxerr.f
+++ CAMx/camxerr.f
@@ -9,7 +9,7 @@
 c     ENVIRON International Corporation
 c             
 c     Modifications:   
-c        none 
+c        Replaced stop with call exit(1) , daniel.oderbolz@psi.ch (2009-06-28)
 c 
 c     Input arguments:
 c        none 
@@ -101,6 +101,7 @@
       write(iout,*) 
       write(iout,*) 
 c
-      stop
+c call exit(1) sets a return value of 1
+      call exit(1)
 c
       end
