--- Makefile
+++ Makefile
@@ -1,6 +1,9 @@
 #------------------------------------------------------------
 #            This is the Makefile for CAMx v4.50
 #
+#     Modified by Daniel Oderbolz for use with the CAMxRunner Installer
+#     Only changed HDF_LIB and HDF_INC
+#
 #     Syntax is: "make platform <DOMAIN=string> <HDF=true>"
 #
 #     For example:
@@ -52,8 +55,8 @@
 MODULES   = -I${MOD_DIR} -module ${MOD_DIR}
 
 ifeq ($(HDF),true)
-   HDF_LIB  = -L/usr/lib -L/usr/local/lib -lhdf5 -lz
-   HDF_INC  = /usr/local/include/hdf5
+   HDF_LIB  = -L/usr/lib -L/usr/local/lib  -L${MYLIBDIR}/hdf -lhdf5 -lz
+   HDF_INC  = ${MYLIBDIR}/hdf/include
    DEFHDF   = -DUSEHDF
    INCLUDES = -I$(CAMx_INC) -I$(IOHDF) -I$(HDF_INC)
    HDF_OBJ  = \
@@ -112,7 +115,7 @@
 	@echo ' '
 
 clean:	
-	rm -f $(OBJCTS) $(IOHDF)/*.o $(MOD_DIR)/*.mod *.a
+	-rm -f $(OBJCTS) $(IOHDF)/*.o $(MOD_DIR)/*.mod *.a
 
 dec:
 	@rm -f $(IOHDF)/*.o $(MOD_DIR)/*.mod
