--- Makefile	Tue Nov 10 09:58:08 2009
+++ Makefile	Fri Nov 13 11:33:41 2009
@@ -201,11 +201,11 @@
            HDF=$(HDF) \
            FC=xlf \
            CC=cc \
-           FLGS="$(INCLUDES) -DIBM $(DEFHDF) -O2 -static -brename:.flush,.flush_ -brename:.etime,.etime_ -brename:.dtime,.dtime_ -brename:.fdate,.fdate_"
+           FLGS="$(INCLUDES) -DIBM $(DEFHDF) -O2 -brename:.flush,.flush_ -brename:.etime,.etime_ -brename:.dtime,.dtime_ -brename:.fdate,.fdate_"
 	make model \
            FC=xlf \
            TARGT=CAMx.$(DOMAIN).ibm \
-           FLGS="-I$(CAMx_INC) -O2 -static -brename:.flush,.flush_ -brename:.etime,.etime_ -brename:.dtime,.dtime_ -brename:.fdate,.fdate_"
+           FLGS="-I$(CAMx_INC) -O2 -brename:.flush,.flush_ -brename:.etime,.etime_ -brename:.dtime,.dtime_ -brename:.fdate,.fdate_"
 
 pg_linux:
 	@rm -f $(IOHDF)/*.o $(MOD_DIR)/*.mod
@@ -220,7 +220,7 @@
 	make model \
            FC=pgf90 \
            TARGT=CAMx.$(DOMAIN).pg_linux \
-           FLGS="-I$(CAMx_INC) -O2 -pc 64 -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic"
+           FLGS="-I$(CAMx_INC) -O2 -pc 64 -Mdalign -Mextend -Mnoframe -byteswapio"
 
 pg_linuxomp:
 	@rm -f $(IOHDF)/*.o $(MOD_DIR)/*.mod
@@ -235,7 +235,7 @@
 	make model \
            FC=pgf90 \
            TARGT=CAMx.$(DOMAIN).pg_linuxomp \
-           FLGS="-I$(CAMx_INC) -O2 -mp -pc 64 -Mdalign -Mextend -Mnoframe -byteswapio -Bstatic"
+           FLGS="-I$(CAMx_INC) -O2 -mp -pc 64 -Mdalign -Mextend -Mnoframe -byteswapio"
 
 i_linux:
 	@rm -f $(IOHDF)/*.o $(MOD_DIR)/*.mod
@@ -246,11 +246,11 @@
            HDF=$(HDF) \
            FC=ifort \
            CC=gcc \
-           FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -align dcommons -extend_source -convert big_endian -ipo -static"
+           FLGS="$(INCLUDES) -DPC_LINUX1 $(DEFHDF) -O2 -align dcommons -extend_source -convert big_endian -ipo"
 	make model \
            FC=ifort \
            TARGT=CAMx.$(DOMAIN).i_linux \
-           FLGS="-I$(CAMx_INC) -O2 -align dcommons -extend_source -convert big_endian -ipo -static"
+           FLGS="-I$(CAMx_INC) -O2 -align dcommons -extend_source -convert big_endian -ipo"
 
 i_linuxomp:
 	@rm -f $(IOHDF)/*.o $(MOD_DIR)/*.mod
