--- airconv.f	2008-05-23 11:10:08.281836951 +0200
+++ ../airconv.f	2008-05-23 10:37:17.453192249 +0200
@@ -7,26 +7,39 @@
 C      OUTPUT ON UNIT 9
 C      DIAGNOSTICS ON UNIT 8
 C
+C   Slightly changed by Daniel Oderbolz for non-interactive operation
+C
+C   Call it like this: airconv source dest type log
+C   e.g:               airconv terrain_domain1_bx3_lucamx.asc terrain_domain1_bx3_lucamx.asc TERRAIN terrain_domain1_bx3_lucamx.log
+
       INCLUDE 'nampar.cmd'
       COMMON /LCM/ EMOB (MXX,MXY,MXSPC)
-      CHARACTER*80 IPATH1,IPATH2,IPATH3,IZERO
+      CHARACTER*255 IPATH1,IPATH2,IPATH3,IZERO
       CHARACTER*10 INFILE
       IZERO='0'
-      WRITE (*,101)
-      READ (*,105) INFILE
-      WRITE (*,102)
-      READ (*,100) IPATH1
-      WRITE (*,103)
-      READ (*,100) IPATH2
-      WRITE (*,104)
-      READ (*,100) IPATH3
-  100 FORMAT (A)
-  101 FORMAT ('ENTER FILE TYPE TO BE CONVERTED (USE AIRSHED CONVENTION)
-     1')
-  102 FORMAT ('ENTER FORMATTED INPUT FILE TO BE CONVERTED')
-  103 FORMAT ('ENTER NAME OF BINARY OUTPUT FILE')
-  104 FORMAT ('ENTER NAME FOR DIAGNOSTIC FILE (0 FOR TTY OUTPUT)')
-  105 FORMAT (A)
+      
+c---  Read arguments   
+c---  This is pgf77 specific code!     
+      call getarg(1,IPATH1)
+      call getarg(2,IPATH2)
+      call getarg(3,INFILE)
+      call getarg(4,IPATH3)      
+      
+C      WRITE (*,101)
+C      READ (*,105) INFILE
+C      WRITE (*,102)
+C      READ (*,100) IPATH1
+C      WRITE (*,103)
+C      READ (*,100) IPATH2
+C      WRITE (*,104)
+C      READ (*,100) IPATH3
+C  100 FORMAT (A)
+C  101 FORMAT ('ENTER FILE TYPE TO BE CONVERTED (USE AIRSHED CONVENTION)
+C     1')
+C  102 FORMAT ('ENTER FORMATTED INPUT FILE TO BE CONVERTED')
+C  103 FORMAT ('ENTER NAME OF BINARY OUTPUT FILE')
+C  104 FORMAT ('ENTER NAME FOR DIAGNOSTIC FILE (0 FOR TTY OUTPUT)')
+C  105 FORMAT (A)
 C
       OPEN (7,FILE=IPATH1)
       OPEN (9,FORM='UNFORMATTED',FILE=IPATH2)
